lsp-methods.el

```diff
@@ -819,19 +829,127 @@ directory."
     (cl-notany (lambda (p) (string-match-p p root))
       lsp-project-blacklist)))
 
+(defun lsp--workspace-folders-capability-p ()
+  "Return whether current server supports workspace folders."
+  (when-let (workspace (gethash "workspace" (lsp--server-capabilities)))
+    (when-let (capability (gethash "workspaceFolders" workspace))
```

Comment:
`when-let*` ?

Reply:
Just checked out when-let* is not available in emacs 25(unless I am mistaken).